### ISSUE TITLE
[ACL] Incorrect behavior for create ACL table binding to VLAN

### ISSUE THREAD

**Issue by ezio-chen** [sonic-net/sonic-utilities#1753]
#### Description

In https://github.com/Azure/sonic-utilities/pull/1475, binding to VLAN will be expanded into VLAN member.

I think the behavior is incorrectly.
If you change the VLAN member ports, the new member will not be restricted by ACL.
Or the member port had join multiple VLANs, other VLAN traffic will also be affected.

I think user shall bind the ACL to member ports and filter VLAN ID in rules if the platforms do not support binding to the VLAN directly.

**Comment by qiluo-msft** [comment]
> Or the member port had join multiple VLANs, other VLAN traffic will also be affected

This is not true. When vlan acl table bind to vlan member port, we adapt the acl rules in the table, ie. add a vlan id rule into the table, so the behavior should be the same.

**Comment by qiluo-msft** [comment]
@ganglyu for awareness.
